### PR TITLE
Add it translation +  add translation keys for navigation tree 'no co…

### PR DESCRIPTION
**Pull request by fracasso**
Add it translation +  add translation keys for navigation tree 'no connection'

### CODE REVIEW

File: webapp/packages/core-app/src/NavigationTree/NavigationTree.tsx [hunk 1]
```diff
@@ -108,8 +113,8 @@ export const NavigationTree = observer(function NavigationTree() {
         emptyPlaceholder={() => styled(navigationTreeStyles)(
           <center>
             <message>
-              No connections.<br />
-              Use the top menu to setup connection to your database.
+              {translate('ui_no_connection')}<br />
```
Comment (devnaumov) [inline webapp/packages/core-app/src/NavigationTree/NavigationTree.tsx]: Better use one token and break it with a \n symbol.

File: webapp/packages/core-localization/src/locales/en.ts [hunk 2]
```diff
@@ -83,4 +83,7 @@ export const defaultENLocale = [
   ['app_root_session_expired_reload', 'Reload'],
   ['app_root_quota_exceeded', 'Quota exceeded'],
   ['core_eventsLog_dbeaverErrorDetails', 'Error details'],
+
+  ['ui_no_connection', "No Connections."],
```
Comment (devnaumov) [inline webapp/packages/core-localization/src/locales/en.ts]: It's not a UI specific so better move it to the core-app package and name the token something like app_navigationTree_no_connections_placeholder

File: webapp/packages/core-app/src/NavigationTree/NavigationTree.tsx [hunk 3]
```diff
@@ -28,6 +28,9 @@ import { navigationTreeDuplicateFilter } from './navigationTreeDuplicateIdFilter
 import { NavigationTreeService } from './NavigationTreeService';
 import { useNavigationTree } from './useNavigationTree';
 
+import { useTranslate } from '@cloudbeaver/core-localization';
+
```
Comment (devnaumov) [inline webapp/packages/core-app/src/NavigationTree/NavigationTree.tsx]: One indent

**Review by devnaumov — CHANGES_REQUESTED** [review]

**Comment by devnaumov** [comment]
Hi @fracasso! Thanks for your contribution. We left some comments that need to be fixed before the merge.

**Comment by LonwoLonwo** [comment]
Hello @fracasso 

Thanks for your contribution! 
Sorry that we missed it.
Unfortunately, I can't review your localization part because it is under all other CloudBeaver Team commits.
So, please create a new PR on the latest devel branch. And we will review it in time.
I need to close this PR due to its unreadability.

**Pull request closed without merging**